src/main/groovy/de/dkfz/roddy/execution/cluster/pbs/PBSCommand.groovy

```diff
@@ -147,6 +147,10 @@ class PBSCommand extends Command {
 
         StringBuilder qsubCall = new StringBuilder(EMPTY)
 
+        if (job.getToolScript()) {
```

Comment:
As discussed, it works with single quotes.
I also added escaping for single quotes (see https://stackoverflow.com/a/20053121), which seems easier and more robust than heredocs.